packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { InputMask } from "./InputMask";
+import { CommonFormFieldProps, FormField, FormFieldProps } from "../FormField";
+
+interface InputPhoneNumberProps
+  extends Omit<CommonFormFieldProps, "align">,
+    Pick<
+      FormFieldProps,
+      | "autocomplete"
+      | "onEnter"
+      | "onFocus"
+      | "onBlur"
+      | "validations"
+      | "readonly"
+      | "prefix"
+      | "suffix"
+    > {
+  readonly value: string;
+  readonly onChange: (value: string) => void;
+
+  /**
+   * Shows a "required" validation message when the component is left empty.
+   */
+  readonly required?: boolean;
```

Comment:
Surprised that we don't do this and force people to write their own message all the time. Like if we can have min/max that adds validation, why shouldn't we do this?

Reply:
Yeah, I like it - I wonder if the "automatic" validation error message is why we didn't do it (see my comment below). Let's see how it works here and then move the pattern over to other inputs if it works?

Reply:
It adds a validation https://github.com/GetJobber/atlantis/blob/2fc03fc89ff6ef3f187f75da59c8bfaadf40f7a8/packages/components/src/InputNumber/InputNumber.tsx#L67-L85

Think of it as a super quick consistent messaging thing for everything and you only ever reach for validation prop when you're doing some crazy validations

Reply:
Ahh then ignore my comment